tools/torchfix/README.md

```diff
@@ -48,3 +48,19 @@ To enable them, use standard flake8 configuration options for the plugin mode or
 
 If you encounter a bug or some other problem with TorchFix, please file an issue on
 https://github.com/pytorch/test-infra/issues, mentioning [TorchFix] in the title.
+
+
+## Rules
+
+### TOR001 Use of removed function
+
+#### torch.solve
+
+This function was deprecated since PyTorch version 1.9 and is now removed.
+
+`torch.solve` is deprecated in favor of `torch.linalg.solve`.
```

Comment:
Seems like we still mention `torch.solve` in our docs. Could you update that ref too please?

https://pytorch.org/docs/stable/complex_numbers.html
<img width="886" alt="image" src="https://github.com/pytorch/test-infra/assets/4468967/182d5e4c-076e-490d-9ffc-a2a0e5781d48">

Reply:
Good find! I'll update that.

Reply:
https://github.com/pytorch/pytorch/pull/113391